hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/metadata/MetadataAddingMessageConverter.java

```diff
@@ -0,0 +1,23 @@
+package pl.allegro.tech.hermes.frontend.publishing.metadata;
+
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.message.wrapper.MessageContentWrapperDispatcher;
+import pl.allegro.tech.hermes.frontend.publishing.message.Message;
+
+import javax.inject.Inject;
+
+public class MetadataAddingMessageConverter {
```

Comment:
We need to invoke this code, question is where? I don't want to move it to PublishingServlet so I decided to not change it.